File: apps/customer-service-admin/src/api/customer/Customer.ts
import { Company } from "../company/Company";

export type Customer = {
  company?: Company | null;
  createdAt: Date;
  email: string | null;
  firstName: string | null;
  id: string;
  lastName: string | null;
  phoneNumber: string | null;
  updatedAt: Date;
};

File: apps/sample-service/src/APIs/Customer/Dtos/Customer.cs
namespace SampleService.APIs.Dtos;

public class Customer
{
    public DateTime CreatedAt { get; set; }

    public string? Email { get; set; }

    public string Id { get; set; }

    public string? Name { get; set; }

    public List<string>? Orders { get; set; }

    public string? Phone { get; set; }

    public DateTime UpdatedAt { get; set; }
}
